app/scss/modal.scss

```diff
@@ -0,0 +1,21 @@
+.overlay {
+  background: rgba(0,0,0,0.6);
+  position: fixed;
+  z-index: 500;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: none;
+}
+#modal {
+  display: none;
+  background: #ffffff;
+  width: 600px;
+  position: fixed;
+  top: 30px;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 15px;
+  z-index: 501;
```

Comment:
is that necessary?

Reply:
looked into it, but I'm not sure if it needs to be 501.